pkg/util/metrics/framework/counter.go

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
```

Comment:
why do we add this to k/k's pkg/util? Would very much like to see a place somewhere in the staging repos. I understand that this will be used by all components? Would component-base be a good place?

Reply:
Sure that sounds reasonable to me. Here's #77353 for creating the sig-instrumentation directory.